test-suite/Makefile

```
@@ -262,6 +247,24 @@ summary.log:
 report: summary.log
 	$(HIDE)bash report.sh
 
+#######################################################################
+# Test suite configuration, to eventually go away
+#######################################################################
+
+# EGJA: using $(ROOT)/_build/default/... here confuses make due to quotes...
+ifeq ($(INSIDE_DUNE),)
+TESTCONFIGURE:=../_build/default/test-suite/tools/coq_config_to_make.exe
+else
+TESTCONFIGURE:=./tools/coq_config_to_make.exe
+endif
+
+$(TESTCONFIGURE): tools/coq_config_to_make.ml
+	dune build ./tools/coq_config_to_make.exe
```

```suggestion
ifeq ($(INSIDE_DUNE),)
TESTCONFIGURE:=../_build/default/test-suite/tools/coq_config_to_make.exe
else
TESTCONFIGURE:=./tools/coq_config_to_make.exe

$(TESTCONFIGURE): tools/coq_config_to_make.ml
	dune build ./tools/coq_config_to_make.exe
endif
```
we shouldn't need to build it from inside dune IIUC

---

Yes, we can do that if you prefer. Indeed there are many options here, happy to follow the last one.

---

Yes, depending on the .inc directly makes the most sense to me

---

Done, still we have a bit of nuissance here if the file is in tree, dune will complain about having the built-in rule and our rule generating the file.

---

Yes, tho I am generating this file now in _build, so the problem is less.

Anyways IMVVHO we should finish Ali PR, we can get together one day or something. Even if not perfect, IMO the model is much better.